ffi/framebuffer.lua

```diff
@@ -22,6 +22,7 @@ local fb = {
     night_mode = false,
     hw_dithering = false, -- will be setup via setupDithering @ startup by reader.lua
     sw_dithering = false, -- will be setup via setupDithering @ startup by reader.lua
+    swipe_animations = false, --will be toggled at page turn by the reader
```

Comment:
Me, _the reader_, will not toggle it at page turn ! But _reader.lua_ will probably be more obedient :)
(Also, space after `--`)